src/frequenz/sdk/timeseries/formula_engine/_formula_engine.py

```diff
@@ -203,11 +203,68 @@ class FormulaEngine(
         QuantityT,
     ],
 ):
-    """
-    The FormulaEngine evaluates formulas and streams the results.
+    """[`FormulaEngine`][frequenz.sdk.timeseries.formula_engine.FormulaEngine]s are a
+    part of the SDK's data pipeline, and provide a way for the SDK to apply formulas on
+    resampled data streams.
 
-    Use the `FormulaBuilder` to create `FormulaEngine` instances.
-    """
+    They are used in the SDK to calculate and stream metrics like
+    [`grid_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.grid_power],
+    [`consumer_power`][frequenz.sdk.timeseries.logical_meter.LogicalMeter.consumer_power],
+    etc., which are building blocks of the
+    [Frequenz SDK Microgrid Model][frequenz.sdk.microgrid--frequenz-sdk-microgrid-model].
+
+    The SDK creates the formulas by analysing the configuration of components in the
+    {{glossary("Component Graph")}}.
+
+    ### Streaming Interface
+
+    The
+    [`FormulaEngine.new_receiver()`][frequenz.sdk.timeseries.formula_engine.FormulaEngine.new_receiver]
+    method can be used to create a
+    [Receiver](https://frequenz-floss.github.io/frequenz-channels-python/latest/reference/frequenz/channels/#frequenz.channels.Receiver)
+    that streams the [Sample][frequenz.sdk.timeseries.Sample]s calculated by the formula
+    engine.
+
+    ```python
+    from frequenz.sdk import microgrid
+
+    battery_pool = microgrid.battery_pool()
+
+    async for power in battery_pool.power.new_receiver():
+        print(f"{power=}")
+    ```
+
+    ### Composition
+
+    Composite `FormulaEngine`s can be built using arithmetic operations on
+    `FormulaEngine`s streaming the same type of data.
+
+    For example, if you're interested in particular composite metric that can be
```

Comment:
```suggestion
    For example, if you're interested in a particular composite metric that can be
```

Reply:
fixed in: https://github.com/frequenz-floss/frequenz-sdk-python/commit/e71612e4